.nvmrc

```diff
@@ -1 +1 @@
-14
```

Comment:
what's this file? It shows empty changes

Reply:
it changed from 14-18, no? it's what node version manager (nvm) uses to determine the version of node to use (see **[here](https://github.com/nvm-sh/nvm#nvmrc))**

Reply:
oops sorry, somehow it showed up as empty file. i see the changes now after refreshing